azure-devops/azext_devops/dev/team/user.py

```diff
@@ -67,8 +68,11 @@ def update_user_entitlement(user, license_type, organization=None, detect=None):
     if '@' in user:
         user = resolve_identity_as_id(user, organization)
     client = get_member_entitlement_management_client(organization)
-    user_entitlement_update = client.update_user_entitlement(document=patch_document, user_id=user)
-    return user_entitlement_update.user_entitlement
+    try:
+        user_entitlement_update = client.update_user_entitlement(document=patch_document, user_id=user)
+        return user_entitlement_update.user_entitlement
+    except Exception:
```

Comment:
not very sure if we should do a blanket catch and convert it to 'Invalid license type.'.
I mean that the exception here can be anything (maybe a bug on service side)

Reply:
Currently, there is an issue in deserialization of response in the client. I''ll raise a bug, and once that fix goes in the client, i shall update the exception logic in our code.

Reply:
In that case should we throw some generic exception rather than a specific exception which could be misleading?

Reply:
Yeah, but then user won't know what the issue is. I think this is fine as its a temporary and once the actual issue is fixed in python client we can throw a specific error in case of invalid access level and a generic one otherwise.